Modules/Filtering/ImageGrid/include/itkResampleImageFilter.hxx

```diff
@@ -540,6 +539,8 @@ ResampleImageFilter< TInputImage, TOutputImage, TInterpolatorPrecisionType, TTra
       // Input requested region is partially outside the largest possible region.
       //   or
       // Input requested region is completely inside the largest possible region.
+      inputRequestedRegion.PadByRadius( m_Interpolator->GetRadius() );
```

Comment:
Concerning the linear interpolator I'd say that the former code would assume a radius of `0` for the padding but with 
https://github.com/InsightSoftwareConsortium/ITK/pull/668/commits/17d23c4f0a3d31c89f11d2bf402f477e7c0c07fd#diff-b143a8bc7c8f5f43d436547ffdb53fddR110
this now extents the requested region by `1`. 
So I assume the failure of `itkResampleImageTest7` originates from this padding but it is not clear to me why.

Reply:
The cover region is now bigger by one, so the indices are wider by 1 and size is greater by 2.